.github/workflows/test-workflow.yml

```diff
@@ -9,9 +9,10 @@ on:
 
 env:
   RELAY_IMAGE_TAG: 0.23.3
-  NETWORK_NODE_IMAGE_TAG: 0.39.0-alpha.0
-  HAVEGED_IMAGE_TAG: 0.39.0-alpha.0
-  MIRROR_IMAGE_TAG: 0.81.0-rc1
+  NETWORK_NODE_IMAGE_TAG: 0.39.0
+  HAVEGED_IMAGE_TAG: 0.39.0
+  MIRROR_IMAGE_TAG: 0.84.0
+  HBAR_RATE_LIMIT_DURATION: 1000
```

Comment:
@quiet-node try moving this to `local.env` and seeing if it gets picked up

Reply:
@quiet-node 
Actually i figured out the issue and opened a ticket on local node.
They resolved it and pushed a new version 2.11.1
Can you bump to use that version and I don't think you'll need to set `HBAR_RATE_LIMIT_DURATION` in that case as rate limiting was recently turned off

Reply:
@Nana-EC Yeah I was able to upgrade the `local node` to 2.11.1 and saw the updated images in docker. However, it seems to fail many of the unit test cases, and most of them are related to `gas`. The full log is provided at the end. Additionally, I attempted to switch the branch to main, which uses `local node@^2.8.0`, but it resulted in failing all the test cases with this error.
```
Error: nonce has already been used [ See: https://links.ethers.org/v5-errors-NONCE_EXPIRED ] (error={"name":"ProviderError","_stack":"ProviderError: [Request ID: e62efeaf-c85c-4955-badc-878a0d37df48] Nonce too low\n    at HttpProvider.request (/Users/user/Development/hedera/hedera-smart-contracts/node_modules/hardhat/src/internal/core/providers/http.ts:88:21)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at EthersProviderWrapper.send (/Users/user/Development/hedera/hedera-smart-contracts/node_modules/@nomiclabs/hardhat-ethers/src/internal/ethers-provider-wrapper.ts:13:20)","code":32001,"_isProviderError":true}, method="sendTransaction", transaction=undefined, code=NONCE_EXPIRED, version=providers/5.7.2)
```

Subsequently, I tried switching the network to 'testnet' just out of curioscity, and surprisingly, all the test cases passed flawlessly without any issues.

Do you think the update to the `local node` causes conflicts with this repository somehow?

And here is the full log of the test with `local node@2.11.1`
```
DiamondTest
duplicate definition - DiamondCut((address,uint8,bytes4[])[],address,bytes)
    1) "before all" hook for "should have three facets -- call to facetAddresses function"

  ERC1155 tests
    2) "before all" hook for "should be able to execute uri(uint256) and returns the same URI for all token types"

  ERC20 tests
    3) "before all" hook for "should be able to execute name()"

  ERC20ExtensionsMock tests
    4) "before all" hook in "ERC20ExtensionsMock tests"

  ERC721 tests
    5) "before all" hook for "should be able to execute name()"

  HIP583 Test Suite
    Direct Ethereum Tx
      Positive
        HBAR Test
          ✔ should be able to create hollow account and transfer HBARs (3084ms)
          ✔ should be able to make second HBARs transfer (5101ms)
          ✔ should be able to make HBARs transfer and sign it with hollow account (6626ms)
        Fungible Token Test
          ✔ should create hollow account and transfer Fungible Tokens (853ms)
          6) should be able to make second Fungible Tokens transfer
          7) should be able to make Fungible Tokens transfer and sign with hollow account
        Non-Fungible Token Test
          ✔ should create hollow account and transfer Non-Fungible Token (340ms)
          8) should be able to make second Non-Fungible Token transfer
          9) should be able to make Non-Fungible Token transfer and sign it with hollow account
      Negative
        ✔ shouldn't be able to get balance of hollow account with no prior transfer
        ✔ shouldn't be able to make transfer from hollow account with no prior transfer (6103ms)

  HIP583 Test Suite - Contract Transfer TX
    10) "before all" hook in "HIP583 Test Suite - Contract Transfer TX"

  HIP583 Test Suite - Ethereum Transfer TX via Precompile
    Fungible Token Test
      11) "before all" hook for "should test that hollow account is created and the amount of fungible tokens is correctly transferred via precompile"
    Non-Fungible Token Test
      12) "before all" hook for "should test that hollow account is created and the amount of non-fungible tokens is correctly transferred via precompile"

  ERC20Contract Test Suite
    13) "before all" hook for "should be able to get token name"

  IERC20 Test Suite
    14) "before all" hook for "should be able to get token name"

  ERC721Contract Test Suite
    15) "before all" hook for "should be able to get token name"

  HRC tests
hrcContract:  0x642e9dCa5383B875A849760D012d0FB42885B7f2
signer:  0x05FbA803Be258049A27B820088bab1cAD2058871
tokenAddress:  0x00000000000000000000000000000000000004ba
    ✔ should be able to associate() to the token from a contract (1559ms)
    - should be able to disssociate() to the token from a contract
    ✔ should be able to associate() to the token from an EOA (804ms)
    - should be able to dissociate() to the token from an EOA
    - should be able to execute associate() via redirectForToken
    - should be able to execute dissociate() via redirectForToken

  RedirectForToken Test Suite
    16) "before all" hook for "should be able to execute name()"

  TokenCreateContract Test Suite
    17) "before all" hook for "should be able to execute burnToken"

  TokenManagmentContract Test Suite
    18) "before all" hook for "should be able to delete token"

  TokenQueryContract Test Suite
    19) "before all" hook for "should query allowance"

  TokenTransferContract Test Suite
    20) "before all" hook for "should NOT be able to use transferFrom on fungible tokens without approval"

  Multicall Test Suite
    static calls with large input
      ✔ should be able to aggregate 10 calls to processLongInput
      ✔ should be able to aggregate 1000 calls to processLongInput (2010ms)
      ✔ should NOT be able to aggregate 5000 calls to processLongInput (820ms)
      21) should be able to aggregate 11 calls to processLongInput and handles a revert
    static calls with large output
      ✔ should be able to aggregate 10 calls to processLongOutput and handle 13 kb of output data (1532ms)
      ✔ should be able to aggregate 80 calls to processLongOutput and handle 820 kb of output data (1727ms)
      ✔ should NOT be able to aggregate 100 calls to processLongOutput (1593ms)
    payable calls with large input
      ✔ should be able to aggregate 10 calls to processLongInputTx (1561ms)
      ✔ should be able to aggregate 130 calls to processLongInputTx (12290ms)
      ✔ should NOT be able to aggregate 150 calls to processLongInputTx (581ms)
    executes multiple state-changing methods
      ✔ should be able to aggregate 10 calls to processLongOutputTx (1579ms)

  Native Precompiles - Ecrecover
    ✔ should be able to call callEcrecover
    22) should be able to call call0x1
    ✔ should not be able to call call0x1 with value (1558ms)
    ✔ should not be able to call send0x1 with no value (1554ms)
    ✔ should not be able to call send0x1 with value (1559ms)
    ✔ should not be able to call transfer0x1 with no value (1549ms)
    ✔ should not be able to call transfer0x1 with value (1550ms)

  Proxy Upgrade Contracts Test Suite
    DEX Upgradable Contract Test Suite
      23) "before all" hook for "should deposit, buy and sell tokens from ExchangeV1"
    Counter Upgradable Contract Test Suite
      24) "before all" hook for "should be able to increase and decrease counter on V1"

  SafeHTS library Test Suite
    25) should be able to get token info
    26) should be able to get fungible token info
    27) should be able to get Non fungible token info
    28) should be able to transfer tokens and hbars atomically

  ShanghaiOpcodes tests
    29) "before all" hook for "should be able to execute opShl()"


  25 passing (5m)
  4 pending
  29 failing

  1) DiamondTest
       "before all" hook for "should have three facets -- call to facetAddresses function":
     ProviderError: [Request ID: c9025e6f-145f-46d0-9f24-3a3f7b78a1f8] Transaction gas limit provided '21000' is insufficient of intrinsic gas required '220456'
      at HttpProvider.request (/Users/user/Development/hedera/hedera-smart-contracts/node_modules/hardhat/src/internal/core/providers/http.ts:88:21)
      at processTicksAndRejections (node:internal/process/task_queues:95:5)
      at EthersProviderWrapper.send (/Users/user/Development/hedera/hedera-smart-contracts/node_modules/@nomiclabs/hardhat-ethers/src/internal/ethers-provider-wrapper.ts:13:20)

  2) ERC1155 tests
       "before all" hook for "should be able to execute uri(uint256) and returns the same URI for all token types":
     ProviderError: [Request ID: 7119bbe5-c1cc-4d2a-a0b7-d360ec9f5e78] Transaction gas limit provided '21000' is insufficient of intrinsic gas required '274888'
      at HttpProvider.request (/Users/user/Development/hedera/hedera-smart-contracts/node_modules/hardhat/src/internal/core/providers/http.ts:88:21)
      at processTicksAndRejections (node:internal/process/task_queues:95:5)
      at EthersProviderWrapper.send (/Users/user/Development/hedera/hedera-smart-contracts/node_modules/@nomiclabs/hardhat-ethers/src/internal/ethers-provider-wrapper.ts:13:20)

  3) ERC20 tests
       "before all" hook for "should be able to execute name()":
     ProviderError: [Request ID: 843d9eba-050f-4195-9617-c976a475b7b6] Transaction gas limit provided '21000' is insufficient of intrinsic gas required '160936'
      at HttpProvider.request (/Users/user/Development/hedera/hedera-smart-contracts/node_modules/hardhat/src/internal/core/providers/http.ts:88:21)
      at processTicksAndRejections (node:internal/process/task_queues:95:5)
      at EthersProviderWrapper.send (/Users/user/Development/hedera/hedera-smart-contracts/node_modules/@nomiclabs/hardhat-ethers/src/internal/ethers-provider-wrapper.ts:13:20)

  4) ERC20ExtensionsMock tests
       "before all" hook in "ERC20ExtensionsMock tests":
     ProviderError: [Request ID: 52cb3f3a-1130-40c6-b111-9cd1fef62242] Transaction gas limit provided '21000' is insufficient of intrinsic gas required '174184'
      at HttpProvider.request (/Users/user/Development/hedera/hedera-smart-contracts/node_modules/hardhat/src/internal/core/providers/http.ts:88:21)
      at processTicksAndRejections (node:internal/process/task_queues:95:5)
      at EthersProviderWrapper.send (/Users/user/Development/hedera/hedera-smart-contracts/node_modules/@nomiclabs/hardhat-ethers/src/internal/ethers-provider-wrapper.ts:13:20)

  5) ERC721 tests
       "before all" hook for "should be able to execute name()":
     ProviderError: [Request ID: b94042ce-badb-4419-be9e-d31990169f34] Transaction gas limit provided '21000' is insufficient of intrinsic gas required '240552'
      at HttpProvider.request (/Users/user/Development/hedera/hedera-smart-contracts/node_modules/hardhat/src/internal/core/providers/http.ts:88:21)
      at processTicksAndRejections (node:internal/process/task_queues:95:5)
      at EthersProviderWrapper.send (/Users/user/Development/hedera/hedera-smart-contracts/node_modules/@nomiclabs/hardhat-ethers/src/internal/ethers-provider-wrapper.ts:13:20)

  6) HIP583 Test Suite
       Direct Ethereum Tx
         Positive
           Fungible Token Test
             should be able to make second Fungible Tokens transfer:
     ProviderError: [Request ID: 5e76d917-eb1b-4ed7-87c2-dd08f5e4b35d] Transaction gas limit provided '21000' is insufficient of intrinsic gas required '25224'
      at HttpProvider.request (/Users/user/Development/hedera/hedera-smart-contracts/node_modules/hardhat/src/internal/core/providers/http.ts:88:21)
      at processTicksAndRejections (node:internal/process/task_queues:95:5)
      at EthersProviderWrapper.send (/Users/user/Development/hedera/hedera-smart-contracts/node_modules/@nomiclabs/hardhat-ethers/src/internal/ethers-provider-wrapper.ts:13:20)

  7) HIP583 Test Suite
       Direct Ethereum Tx
         Positive
           Fungible Token Test
             should be able to make Fungible Tokens transfer and sign with hollow account:
     ProviderError: [Request ID: 2268b0b6-5199-4ea7-832f-bd4163dc86d8] Transaction gas limit provided '21000' is insufficient of intrinsic gas required '25224'
      at HttpProvider.request (/Users/user/Development/hedera/hedera-smart-contracts/node_modules/hardhat/src/internal/core/providers/http.ts:88:21)
      at processTicksAndRejections (node:internal/process/task_queues:95:5)
      at EthersProviderWrapper.send (/Users/user/Development/hedera/hedera-smart-contracts/node_modules/@nomiclabs/hardhat-ethers/src/internal/ethers-provider-wrapper.ts:13:20)

  8) HIP583 Test Suite
       Direct Ethereum Tx
         Positive
           Non-Fungible Token Test
             should be able to make second Non-Fungible Token transfer:
     Error: nonce has already been used [ See: https://links.ethers.org/v5-errors-NONCE_EXPIRED ] (error={"name":"ProviderError","_stack":"ProviderError: [Request ID: 05f5688d-39ab-4f36-bb21-37562d9dea7b] Nonce too low. Provided nonce: 165, current nonce: 166\n    at HttpProvider.request (/Users/user/Development/hedera/hedera-smart-contracts/node_modules/hardhat/src/internal/core/providers/http.ts:88:21)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at EthersProviderWrapper.send (/Users/user/Development/hedera/hedera-smart-contracts/node_modules/@nomiclabs/hardhat-ethers/src/internal/ethers-provider-wrapper.ts:13:20)","code":32001,"_isProviderError":true}, method="sendTransaction", transaction=undefined, code=NONCE_EXPIRED, version=providers/5.7.2)
      at Logger.makeError (node_modules/@ethersproject/logger/src.ts/index.ts:269:28)
      at Logger.throwError (node_modules/@ethersproject/logger/src.ts/index.ts:281:20)
      at checkError (node_modules/@ethersproject/providers/src.ts/json-rpc-provider.ts:105:16)
      at /Users/user/Development/hedera/hedera-smart-contracts/node_modules/@ethersproject/providers/src.ts/json-rpc-provider.ts:265:24
      at processTicksAndRejections (node:internal/process/task_queues:95:5)

  9) HIP583 Test Suite
       Direct Ethereum Tx
         Positive
           Non-Fungible Token Test
             should be able to make Non-Fungible Token transfer and sign it with hollow account:

      AssertionError: expected 1 to be above 1. The numerical values of the given "ethers.BigNumber" and "ethers.BigNumber" inputs were compared, and they differed.
      + expected - actual


      at Context.<anonymous> (test/hip-583/HIP583.js:400:41)
      at processTicksAndRejections (node:internal/process/task_queues:95:5)

  10) HIP583 Test Suite - Contract Transfer TX
       "before all" hook in "HIP583 Test Suite - Contract Transfer TX":
     ProviderError: [Request ID: f08a1f8b-8433-4b4f-8971-2942594c0003] Transaction gas limit provided '21000' is insufficient of intrinsic gas required '91016'
      at HttpProvider.request (/Users/user/Development/hedera/hedera-smart-contracts/node_modules/hardhat/src/internal/core/providers/http.ts:88:21)
      at processTicksAndRejections (node:internal/process/task_queues:95:5)
      at EthersProviderWrapper.send (/Users/user/Development/hedera/hedera-smart-contracts/node_modules/@nomiclabs/hardhat-ethers/src/internal/ethers-provider-wrapper.ts:13:20)

  11) HIP583 Test Suite - Ethereum Transfer TX via Precompile
       Fungible Token Test
         "before all" hook for "should test that hollow account is created and the amount of fungible tokens is correctly transferred via precompile":
     ProviderError: [Request ID: 9e85f287-0bb2-4efa-9076-907d02e4c2a9] Transaction gas limit provided '21000' is insufficient of intrinsic gas required '23176'
      at HttpProvider.request (/Users/user/Development/hedera/hedera-smart-contracts/node_modules/hardhat/src/internal/core/providers/http.ts:88:21)
      at processTicksAndRejections (node:internal/process/task_queues:95:5)
      at EthersProviderWrapper.send (/Users/user/Development/hedera/hedera-smart-contracts/node_modules/@nomiclabs/hardhat-ethers/src/internal/ethers-provider-wrapper.ts:13:20)

  12) HIP583 Test Suite - Ethereum Transfer TX via Precompile
       Non-Fungible Token Test
         "before all" hook for "should test that hollow account is created and the amount of non-fungible tokens is correctly transferred via precompile":
     ProviderError: [Request ID: f23d23ad-2554-4dec-b771-424b63d01a8d] Transaction gas limit provided '21000' is insufficient of intrinsic gas required '23176'
      at HttpProvider.request (/Users/user/Development/hedera/hedera-smart-contracts/node_modules/hardhat/src/internal/core/providers/http.ts:88:21)
      at processTicksAndRejections (node:internal/process/task_queues:95:5)
      at EthersProviderWrapper.send (/Users/user/Development/hedera/hedera-smart-contracts/node_modules/@nomiclabs/hardhat-ethers/src/internal/ethers-provider-wrapper.ts:13:20)

  13) ERC20Contract Test Suite
       "before all" hook for "should be able to get token name":
     ProviderError: [Request ID: 23c1a769-6aa1-4045-986e-3658a9a819ab] Transaction gas limit provided '21000' is insufficient of intrinsic gas required '23176'
      at HttpProvider.request (/Users/user/Development/hedera/hedera-smart-contracts/node_modules/hardhat/src/internal/core/providers/http.ts:88:21)
      at processTicksAndRejections (node:internal/process/task_queues:95:5)
      at EthersProviderWrapper.send (/Users/user/Development/hedera/hedera-smart-contracts/node_modules/@nomiclabs/hardhat-ethers/src/internal/ethers-provider-wrapper.ts:13:20)

  14) IERC20 Test Suite
       "before all" hook for "should be able to get token name":
     ProviderError: [Request ID: a79331e1-d4f2-4d7d-b7c3-5eedcc8c62e4] Transaction gas limit provided '21000' is insufficient of intrinsic gas required '23176'
      at HttpProvider.request (/Users/user/Development/hedera/hedera-smart-contracts/node_modules/hardhat/src/internal/core/providers/http.ts:88:21)
      at processTicksAndRejections (node:internal/process/task_queues:95:5)
      at EthersProviderWrapper.send (/Users/user/Development/hedera/hedera-smart-contracts/node_modules/@nomiclabs/hardhat-ethers/src/internal/ethers-provider-wrapper.ts:13:20)

  15) ERC721Contract Test Suite
       "before all" hook for "should be able to get token name":
     ProviderError: [Request ID: 079a3baa-3c32-4f83-b607-c75c8f62dda7] Transaction gas limit provided '21000' is insufficient of intrinsic gas required '23176'
      at HttpProvider.request (/Users/user/Development/hedera/hedera-smart-contracts/node_modules/hardhat/src/internal/core/providers/http.ts:88:21)
      at processTicksAndRejections (node:internal/process/task_queues:95:5)
      at EthersProviderWrapper.send (/Users/user/Development/hedera/hedera-smart-contracts/node_modules/@nomiclabs/hardhat-ethers/src/internal/ethers-provider-wrapper.ts:13:20)

  16) RedirectForToken Test Suite
       "before all" hook for "should be able to execute name()":
     Error: transaction failed [ See: https://links.ethers.org/v5-errors-CALL_EXCEPTION ] (transactionHash="0x0c82d9a3967e6c745d7a7f423a94e516ccf4eef63458f0ff744b97faef830a40", transaction={"hash":"0x0c82d9a3967e6c745d7a7f423a94e516ccf4eef63458f0ff744b97faef830a40","type":2,"accessList":null,"blockHash":"0x27fcea96c167be5ce43a43e50c450f4169cbfa79a4010dc54508467aaa3eb32d","blockNumber":1202,"transactionIndex":7,"confirmations":1,"from":"0x05FbA803Be258049A27B820088bab1cAD2058871","maxFeePerGas":{"type":"BigNumber","hex":"0x59"},"gasLimit":{"type":"BigNumber","hex":"0x0c3500"},"to":null,"value":{"type":"BigNumber","hex":"0x3b9aca00"},"nonce":202,"data":"0x8f74a17b00000000000000000000000005fba803be258049a27b820088bab1cad20588710000000000000000000000000000000000000000000000000000000000000040000000000000000000000000000000000000000000000000000000000000002103c065d7a6c816e36a20460f16bcb91d33a4c0231cccfb648776c392188d56184900000000000000000000000000000000000000000000000000000000000000","r":"0x289d0f15c4286abc1da92941bd5121b5eefa775a20f93a6a9b2bd324226d8651","s":"0x309c6b72614e9d4e162f61cd4a901afdd7d1faa97d89def615572ee8f0149dcd","v":0,"creates":"0x55a8a7ba679e44F8E68659D5DC79D6C1096CF272","chainId":298}, receipt={"to":null,"from":"0x00000000000000000000000000000000000003F5","contractAddress":null,"transactionIndex":7,"gasUsed":{"type":"BigNumber","hex":"0x0c3500"},"logsBloom":"0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000","blockHash":"0x27fcea96c167be5ce43a43e50c450f4169cbfa79a4010dc54508467aaa3eb32d","transactionHash":"0x0c82d9a3967e6c745d7a7f423a94e516ccf4eef63458f0ff744b97faef830a40","logs":[],"blockNumber":1202,"confirmations":1,"cumulativeGasUsed":{"type":"BigNumber","hex":"0x0c3500"},"effectiveGasPrice":{"type":"BigNumber","hex":"0xcf38224400"},"status":0,"type":0,"byzantium":true}, code=CALL_EXCEPTION, version=providers/5.7.2)
      at Logger.makeError (node_modules/@ethersproject/logger/src.ts/index.ts:269:28)
      at Logger.throwError (node_modules/@ethersproject/logger/src.ts/index.ts:281:20)
      at EthersProviderWrapper.<anonymous> (node_modules/@ethersproject/providers/src.ts/base-provider.ts:1549:24)
      at step (node_modules/@ethersproject/providers/lib/base-provider.js:48:23)
      at Object.next (node_modules/@ethersproject/providers/lib/base-provider.js:29:53)
      at fulfilled (node_modules/@ethersproject/providers/lib/base-provider.js:20:58)
      at processTicksAndRejections (node:internal/process/task_queues:95:5)

  17) TokenCreateContract Test Suite
       "before all" hook for "should be able to execute burnToken":
     ProviderError: [Request ID: b0c5a875-4bd9-4d4a-b1a5-c7bc713436e0] Transaction gas limit provided '21000' is insufficient of intrinsic gas required '23176'
      at HttpProvider.request (/Users/user/Development/hedera/hedera-smart-contracts/node_modules/hardhat/src/internal/core/providers/http.ts:88:21)
      at processTicksAndRejections (node:internal/process/task_queues:95:5)
      at EthersProviderWrapper.send (/Users/user/Development/hedera/hedera-smart-contracts/node_modules/@nomiclabs/hardhat-ethers/src/internal/ethers-provider-wrapper.ts:13:20)

  18) TokenManagmentContract Test Suite
       "before all" hook for "should be able to delete token":
     ProviderError: [Request ID: 5fb1f232-1742-4192-ab54-294a687421b7] Transaction gas limit provided '21000' is insufficient of intrinsic gas required '23176'
      at HttpProvider.request (/Users/user/Development/hedera/hedera-smart-contracts/node_modules/hardhat/src/internal/core/providers/http.ts:88:21)
      at processTicksAndRejections (node:internal/process/task_queues:95:5)
      at EthersProviderWrapper.send (/Users/user/Development/hedera/hedera-smart-contracts/node_modules/@nomiclabs/hardhat-ethers/src/internal/ethers-provider-wrapper.ts:13:20)

  19) TokenQueryContract Test Suite
       "before all" hook for "should query allowance":
     ProviderError: [Request ID: ee763e59-9203-4d74-abd1-75af4ab6351b] Transaction gas limit provided '21000' is insufficient of intrinsic gas required '23176'
      at HttpProvider.request (/Users/user/Development/hedera/hedera-smart-contracts/node_modules/hardhat/src/internal/core/providers/http.ts:88:21)
      at processTicksAndRejections (node:internal/process/task_queues:95:5)
      at EthersProviderWrapper.send (/Users/user/Development/hedera/hedera-smart-contracts/node_modules/@nomiclabs/hardhat-ethers/src/internal/ethers-provider-wrapper.ts:13:20)

  20) TokenTransferContract Test Suite
       "before all" hook for "should NOT be able to use transferFrom on fungible tokens without approval":
     ProviderError: [Request ID: 00b57c6a-95eb-4f9d-a7fe-4a8eace82f54] Transaction gas limit provided '21000' is insufficient of intrinsic gas required '23176'
      at HttpProvider.request (/Users/user/Development/hedera/hedera-smart-contracts/node_modules/hardhat/src/internal/core/providers/http.ts:88:21)
      at processTicksAndRejections (node:internal/process/task_queues:95:5)
      at EthersProviderWrapper.send (/Users/user/Development/hedera/hedera-smart-contracts/node_modules/@nomiclabs/hardhat-ethers/src/internal/ethers-provider-wrapper.ts:13:20)

  21) Multicall Test Suite
       static calls with large input
         should be able to aggregate 11 calls to processLongInput and handles a revert:

      AssertionError: expected '0x' to equal '0x08c379a0000000000000000000000000000…'
      + expected - actual

      -0x
      +0x08c379a0000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000174d756c746963616c6c333a2063616c6c206661696c6564000000000000000000

      at Context.<anonymous> (test/multicall/Multicall.js:240:27)
      at processTicksAndRejections (node:internal/process/task_queues:95:5)

  22) Native Precompiles - Ecrecover
       should be able to call call0x1:
     ProviderError: [Request ID: 340ccf04-e107-4ae2-98c0-597f69495289] Transaction gas limit provided '21000' is insufficient of intrinsic gas required '27272'
      at HttpProvider.request (/Users/user/Development/hedera/hedera-smart-contracts/node_modules/hardhat/src/internal/core/providers/http.ts:88:21)
      at processTicksAndRejections (node:internal/process/task_queues:95:5)
      at EthersProviderWrapper.send (/Users/user/Development/hedera/hedera-smart-contracts/node_modules/@nomiclabs/hardhat-ethers/src/internal/ethers-provider-wrapper.ts:13:20)

  23) Proxy Upgrade Contracts Test Suite
       DEX Upgradable Contract Test Suite
         "before all" hook for "should deposit, buy and sell tokens from ExchangeV1":
     ProviderError: [Request ID: a1459645-2660-4f91-b704-8e65c6870d38] Transaction gas limit provided '21000' is insufficient of intrinsic gas required '237288'
      at HttpProvider.request (/Users/user/Development/hedera/hedera-smart-contracts/node_modules/hardhat/src/internal/core/providers/http.ts:88:21)
      at processTicksAndRejections (node:internal/process/task_queues:95:5)
      at EthersProviderWrapper.send (/Users/user/Development/hedera/hedera-smart-contracts/node_modules/@nomiclabs/hardhat-ethers/src/internal/ethers-provider-wrapper.ts:13:20)

  24) Proxy Upgrade Contracts Test Suite
       Counter Upgradable Contract Test Suite
         "before all" hook for "should be able to increase and decrease counter on V1":
     ProviderError: [Request ID: 880d346b-4b2b-459c-aaeb-267af175df00] Transaction gas limit provided '21000' is insufficient of intrinsic gas required '198760'
      at HttpProvider.request (/Users/user/Development/hedera/hedera-smart-contracts/node_modules/hardhat/src/internal/core/providers/http.ts:88:21)
      at processTicksAndRejections (node:internal/process/task_queues:95:5)
      at EthersProviderWrapper.send (/Users/user/Development/hedera/hedera-smart-contracts/node_modules/@nomiclabs/hardhat-ethers/src/internal/ethers-provider-wrapper.ts:13:20)

  25) SafeHTS library Test Suite
       should be able to get token info:
     ProviderError: [Request ID: d8b61d30-750f-428c-8985-df5b6b6873ba] Transaction gas limit provided '21000' is insufficient of intrinsic gas required '22152'
      at HttpProvider.request (/Users/user/Development/hedera/hedera-smart-contracts/node_modules/hardhat/src/internal/core/providers/http.ts:88:21)
      at processTicksAndRejections (node:internal/process/task_queues:95:5)
      at EthersProviderWrapper.send (/Users/user/Development/hedera/hedera-smart-contracts/node_modules/@nomiclabs/hardhat-ethers/src/internal/ethers-provider-wrapper.ts:13:20)

  26) SafeHTS library Test Suite
       should be able to get fungible token info:
     ProviderError: [Request ID: dbad7e25-2429-495e-b538-b1e2d9615cd8] Transaction gas limit provided '21000' is insufficient of intrinsic gas required '22152'
      at HttpProvider.request (/Users/user/Development/hedera/hedera-smart-contracts/node_modules/hardhat/src/internal/core/providers/http.ts:88:21)
      at processTicksAndRejections (node:internal/process/task_queues:95:5)
      at EthersProviderWrapper.send (/Users/user/Development/hedera/hedera-smart-contracts/node_modules/@nomiclabs/hardhat-ethers/src/internal/ethers-provider-wrapper.ts:13:20)

  27) SafeHTS library Test Suite
       should be able to get Non fungible token info:
     ProviderError: [Request ID: b033b413-6b54-4517-ad86-558e048b1744] Transaction gas limit provided '21000' is insufficient of intrinsic gas required '23176'
      at HttpProvider.request (/Users/user/Development/hedera/hedera-smart-contracts/node_modules/hardhat/src/internal/core/providers/http.ts:88:21)
      at processTicksAndRejections (node:internal/process/task_queues:95:5)
      at EthersProviderWrapper.send (/Users/user/Development/hedera/hedera-smart-contracts/node_modules/@nomiclabs/hardhat-ethers/src/internal/ethers-provider-wrapper.ts:13:20)

  28) SafeHTS library Test Suite
       should be able to transfer tokens and hbars atomically:
     ProviderError: [Request ID: e5e8e1bf-8881-4ed7-aa4b-09a9dd57457a] Transaction gas limit provided '21000' is insufficient of intrinsic gas required '23176'
      at HttpProvider.request (/Users/user/Development/hedera/hedera-smart-contracts/node_modules/hardhat/src/internal/core/providers/http.ts:88:21)
      at processTicksAndRejections (node:internal/process/task_queues:95:5)
      at EthersProviderWrapper.send (/Users/user/Development/hedera/hedera-smart-contracts/node_modules/@nomiclabs/hardhat-ethers/src/internal/ethers-provider-wrapper.ts:13:20)

  29) ShanghaiOpcodes tests
       "before all" hook for "should be able to execute opShl()":
     ProviderError: [Request ID: 7627812f-d7ea-456f-954a-26b44a51905f] Transaction gas limit provided '21000' is insufficient of intrinsic gas required '64584'
      at HttpProvider.request (/Users/user/Development/hedera/hedera-smart-contracts/node_modules/hardhat/src/internal/core/providers/http.ts:88:21)
      at processTicksAndRejections (node:internal/process/task_queues:95:5)
      at EthersProviderWrapper.send (/Users/user/Development/hedera/hedera-smart-contracts/node_modules/@nomiclabs/hardhat-ethers/src/internal/ethers-provider-wrapper.ts:13:20)
```

Reply:
I don't see the local node update. https://github.com/hashgraph/hedera-smart-contracts/blob/main/package.json#L20 should be updated to 2.11.1 so see the difference.

Reply:
Right sorry I thought I would fix the local unit testing problem first before push the code. But yeah I can push the changes now. give me a sec